src/Components/SoundUploader/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useRef, useState } from "react";
+import styled from "styled-components";
+
+const FileInput = styled.input``;
+
+const SoundUploader = ({}) => {
+  const inputEl = useRef(null);
+
+  const uploadSound = event => {
+    try {
+      const file = event.target.files[0];
+      if (file.size > 1000000) {
+        // possibly change this to a flash-message for better UX/UI in the future
+        alert("File size must be less than 1MB");
```

Comment:
This is probably a good reason to use `useState`. Instead of doing an `alert` what we can do is keep an `error` state that changes to this message whenever the `file.size` exceeds the limit. I suggest we do away with `alert` messages but instead have a `div` (or I suppose we can use the `Error` component here?) that gets rendered whenever the `error` state isn't `null`. What do you think? :)

Reply:
agree 100%! hence the comment about changing to some sort of flash message, wanted to go over it with the team before actually doing anything concrete the alert's just there for now to ensure that statement is triggering correctly.

i'll fix this up shortly to display in the actual page instead of alert, thanks!